system/sqlstore.go

```diff
@@ -24,19 +26,26 @@ type SQLStore struct {
 
 // NewSQLStore returns a new SQLStore. The keyPairs are used in the same way as
 // the gorilla sessions CookieStore.
-func NewSQLStore(dbMap *gorp.DbMap, keyPairs ...[]byte) *SQLStore {
+func NewSQLStore(ctx context.Context, wg *sync.WaitGroup, dbMap *gorp.DbMap, keyPairs ...[]byte) *SQLStore {
```

Comment:
I think `NewSQLStore` could follow the same pattern as introduced in #528. 

https://github.com/decred/dcrstakepool/blob/a2ca2e88a0dcb71214b43c75a48b573eae3898a1/backend/stakepoold/rpc/rpcserver/context.go#L797-L809

I think deferring the `wg.Done()` and using `return` instead of `break out` would be more consistent with the rest of the code, and it is a lot cleaner IMO.

Reply:
You resolved this one but I dont think the change has been made?

Reply:
I totally overlooked the "break out", thank you for finding and pointing that out.